src/isomorphic/deprecated/__tests__/cloneWithProps-test.js

```diff
@@ -30,6 +30,26 @@ describe('cloneWithProps', function() {
     onlyChild = require('onlyChild');
     cloneWithProps = require('cloneWithProps');
     emptyObject = require('emptyObject');
+    spyOn(console, 'error');
+  });
+
+  it('should warn because it is deprecated', function() {
+    var Grandparent = React.createClass({
+      render: function() {
+        return <Parent><div /></Parent>;
+      },
+    });
+    var Parent = React.createClass({
+      render: function() {
+        return (
+          <div>
+            {cloneWithProps(onlyChild(this.props.children), {})}
+          </div>
+        );
+      },
+    });
+    var component = ReactTestUtils.renderIntoDocument(<Grandparent />);
```

Comment:
Why does Grandparent exist here?  Seems like it should just be ReactTestUtils.renderIntoDocument(<Parent><div /></Parent>);

Also, why are you saving the result as component?  The variable appears to be unused/unnecessary.